### ISSUE TITLE
File.copy: Problem using four-digit chmod with leading 0

### ISSUE THREAD

**Issue by kraileth** [comtrya/comtrya#168]
If using `file.copy` together with a four-digit `chmod` that has a leading zero (e.g. to clear the sticky bit), Comtrya errors out when trying to load the manifest like this:

```
manifest_load{manifest="foo.yaml"}: actions: invalid type: string "0644", expected u32 at line 3 column 3
```

Quoting does not help. Four-digit values without a leading 0 seem to be fine.

Example:

```
- action: file.copy
  from: some-file
  to: /var/testfile
  chmod: 0644
```

Comtrya version used: 0.7.4 on FreeBSD

**Comment by icepuma** [comment]
@kraileth could you please confirm, that the issue is resolved as soon as Rust 1.61.0 is available for FreeBSD

**Comment by kraileth** [comment]
@icepuma Both "644" and "0644" work now; Comtrya will accept either and set the permissions correctly. Thanks for fixing this!
